terraform/secrets.tfvars

```diff
@@ -0,0 +1,2 @@
+aws_access_key = "AKIAWB5UZPEQJWUHFRWL"
```

Comment:
This shouldn't be committed. Please remove.

Reply:
Done